Allow reserved spaces for local competitors in finals (#57)
CarterKoala/wca-regulations-wrc: wca-guidelines.md

- 9f4+) [CLARIFICATION] The result of an attempt should be a DNF if the competitor began the attempt by indicating that they are ready (see [Regulation A3b2](regulations:regulation:A3b2)) but the attempt was disqualified.
- 9f5+) [CLARIFICATION] The result of an attempt is DNS if the competitor was eligible for the attempt and did not start it (see [Regulation A3b2](regulations:regulation:A3b2)). If the competitor did not qualify for an attempt (e.g. in a Cutoff Round), they have no result for the attempt.
- 9p2+) [CLARIFICATION] Result refers to the single or average that determines the ranking of the corresponding round.

> - 9p2b+) [ADDITION] If there are not enough competitors representing the relevant region to fill the reserved spaces, they are replaced with the top ranked competitors representing any region.

Author: ```suggestion
- 9p2b+) [ADDITION] If there are not enough competitors representing the relevant region to fill the reserved spaces, they are replaced with the best competitors representing any region.
```

> - 9p2b++) [RECOMMENDATION] Any use of reserved spaces for final rounds should be clearly announced before the competition.

Author: ```suggestion
- 9p2b++) [RECOMMENDATION] Any use of reserved spaces should be clearly announced before the competition.
```

- 9f4+) [CLARIFICATION] The result of an attempt should be a DNF if the competitor began the attempt by indicating that they are ready (see [Regulation A3b2](regulations:regulation:A3b2)) but the attempt was disqualified.
- 9f5+) [CLARIFICATION] The result of an attempt is DNS if the competitor was eligible for the attempt and did not start it (see [Regulation A3b2](regulations:regulation:A3b2)). If the competitor did not qualify for an attempt (e.g. in a Cutoff Round), they have no result for the attempt.
- 9p2+) [CLARIFICATION] Result refers to the single or average that determines the ranking of the corresponding round.
- 9p2b+) [ADDITION] If there are not enough competitors representing the relevant region to fill the reserved spaces, they are replaced with the best competitors representing any region.

> - 9p2b++) [ADDITION] For a competition designated as both a World Championship and a Continental Championship and/or National Championship, this exception does not apply. For a competition designated as both a Continental Championship and a National Championship, places may only be reserved for competitors representing the relevant continent.

Author: ```suggestion
- 9p2b++) [ADDITION] For a competition designated as both a World Championship and a Continental Championship and/or National Championship, this exception does not apply. For a competition designated as both a Continental Championship and a National Championship, spaces may only be reserved for competitors representing the relevant continent.
```

> - 9p2b+++) [RECOMMENDATION] Any use of reserved spaces for final rounds should be clearly announced before the competition.

Author: ```suggestion
- 9p2b+++) [RECOMMENDATION] Any use of reserved spaces should be clearly announced before the competition.
```
